docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -1097,6 +1097,9 @@ The following view matchers have been introduced in [[= product_name =]] v4.6.0:
 ### Full changelog
 
 [[% include 'snippets/release_46.md' %]]
+
+To update your application, see the [update instructions](update_from_4.6.md).
```

Comment:
let's imagine we just published 4.6.1.
Some people might still be on 4.5.x.
`update_from_4.6.md` doesn't link to `update_from_4.5.md`.
Maybe it should be introduced here. (But maybe it should be introduced in `update_from_4.6.md` and not in `ibexa_dxp_v4.6.md`. For 4.4 to 4.6, the chain is again broken.)
If introduced here, it could be something like this:

```suggestion
To update your application, see the [4.6 update instructions](update_from_4.6.md).

To update your application from v4.5.x, see the [4.5 to 4.6 update instructions](update_from_4.5.md).
```

Reply:
I lean toward adding it to `update_from_4.6.md` - there should be a sentence in the introduction saying that if you're not on 4.6 yet, you should look at the instructions for the older verions.

I'd do it in a follow-up, ok for you? I don't know how far back the chain will go 😅

Reply:
You're right, let wait for this to become needed, we'll know then how it really is.